apps/demo/emails/raycast-magic-link.tsx

```diff
@@ -0,0 +1,126 @@
+import { Button } from '@react-email/button';
+import { Container } from '@react-email/container';
+import { Head } from '@react-email/head';
+import { Hr } from '@react-email/hr';
+import { Html } from '@react-email/html';
+import { Img } from '@react-email/img';
+import { Link } from '@react-email/link';
+import { Preview } from '@react-email/preview';
+import { Section } from '@react-email/section';
+import { Text } from '@react-email/text';
+import * as React from 'react';
+
+export default function Email() {
+  return (
+    <Html>
+      <Head />
+      <Preview>
+        Log in with this magic link.
+      </Preview>
+      <Section style={main}>
+        <Container style={container}>
+          <Img
+            src="https://miro.medium.com/max/1024/1*ThvbH5hKHEM4jpuAqRNfBQ.png"
+            width="170"
+            height="50"
+            alt="Koala"
+            style={logo}
+          />
+          <Container style={container2}>
+            <Text style={heading}>🪄 Your magic link</Text>
+            <Link
+            href="https://www.raycast.com"
+            target="_blank"
+            style={{
+              fontSize: '22px',
+              display: 'block',
+              marginBottom: '16px',
+              marginTop: '50px',
+            }}
+            >
+                👉 Click here to sign in 👈
+            </Link>
+            <Text style={paragraph}>
+                If you didn't request this, please ignore this email.
+            </Text>
+            <Text style={paragraph2}>
+                Best,
+                <br />
+                - Raycast Team
+            </Text>
+            <Hr style={hr} />
+            <Img
+                src="https://miro.medium.com/max/1024/1*ThvbH5hKHEM4jpuAqRNfBQ.png"
```

Comment:
Images must be stored in the assets folder, so we don't rely on images missing in case someone file the files